lib/Cargo.toml

```diff
@@ -1,5 +1,8 @@
 [workspace]
 members = [
+    # Binary generation crate. Breaks nomenclature to avoid collision with uniffi_bindgen library crate.
+    # See https://mozilla.github.io/uniffi-rs/tutorial/foreign_language_bindings.html#multi-crate-workspaces
+    "uniffi-bindgen", 
```

Comment:
Using `uniffi_bindgen` as name for the crate will cause conflict with the CLI library crate:

```bash
$ cargo run -p uniffi_bindgen

error: There are multiple `uniffi_bindgen` packages in your project, and the specification `uniffi_bindgen` is ambiguous.
Please re-run this command with `-p <spec>` where `<spec>` is one of the following:
  uniffi_bindgen@0.1.0
  uniffi_bindgen@0.23.0
```
If we dont want to break the nomenclature, we could call it in another way. Currently following what the guide [recommends](https://mozilla.github.io/uniffi-rs/tutorial/foreign_language_bindings.html#multi-crate-workspaces) regarding the crate name: 

> Name the crate uniffi-bindgen

Not sure is this will be assumed later by UniFFi.

WDYT @hauleth @zduny ?

Reply:
Do what guide recommends (add `uniffi-bindgen` member to workspace).

You can see my [example](https://github.com/zduny/uniffi-example/blob/main/uniffi-bindgen/) for reference.

Please also remember to update (just remove `uniffi-bindgen` installation):
- https://github.com/eigerco/uniffi-zcash-lib/blob/main/.github/workflows/main.yml
- https://github.com/eigerco/uniffi-zcash-lib/blob/main/docker/Dockerfile

Reply:
Hi @zduny, thanks for review. Removed CI step at 00165384a9f4f4727439d16b5f35f866843b7c4f . Moving to final review !